nexus/src/db/datastore.rs

```diff
@@ -282,10 +310,42 @@ impl DataStore {
                     LookupType::ByName(name.as_str().to_owned()),
                 )
             })
+            .map(|o| (authz::FLEET.organization(o.id()), o))
+    }
+
+    /// Look up the id for an organization based on its name
+    ///
+    /// Returns an [`authz::Organization`] (which makes the id available).
+    ///
+    /// This function does no authz checks because it is not possible to know
+    /// just by looking up an Organization's id what privileges are required.
+    pub async fn organization_lookup_id(
```

Comment:
This is the same as the old `organization_lookup()` function, but using the common `organization_lookup_noauthz()` and renamed for clarity.